### DISCUSSION TITLE
There are few CSS mistake in serenity template.

### DISCUSSION THREAD

**Discussion by nithinv86** [primefaces/community#1147] [PrimeNG Templates]
Upon reviewing the Serenity template, I have identified a few CSS mistakes that may need attention. These issues are affecting the visual presentation and overall user experience. Here is a brief summary of the identified problems:

1. src/styles/serenity/sass/layout/_main.scss: line no 692

> &:first-child(even) {} - Why do we need it with the first child?
![Screenshot 2024-02-08 at 5 49 17 PM](https://github.com/primefaces/community/assets/24265744/0ac30817-b898-4450-a9f2-302973a6b513)

2. src/styles/serenity/sass/layout/_landing.scss: Line no 518

> adding-left: 15px; - Typo error P is missing.
![Screenshot 2024-02-08 at 5 49 01 PM](https://github.com/primefaces/community/assets/24265744/d6e44a9d-ceff-439c-aa80-e046436b442c)

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for reporting the issue we'll investigate.
